src/store/Developers.js

```diff
@@ -49,7 +50,23 @@ const actions = {
   }
 };
 
-const getters = {};
+const getters = {
+  newDevelopers(state) {
+    const newDevs = clone(state.developers).filter(dev => {
+      const lastMonthDate = new Date();
+      const ONE_MONTH = 30;
+      lastMonthDate.setDate(lastMonthDate.getDate() - ONE_MONTH);
```

Comment:
May we move this logic (get last month date) before filter function.

Reply:
Done